src/Components/TestExplorer.fs

```diff
@@ -312,13 +313,12 @@ module TrxParser =
         { Message: string option
           StackTrace: string option }
 
-    type Output = { ErrorInfo: ErrorInfo }
```

Comment:
UnitTestResult and it's children (like output) are made to reflect the structure of the Trx file.
To follow that pattern, the new data would be included in this Output object
```fs
type Output = 
  { 
    ErrorInfo: ErrorInfo 
    StdOut: string
  }
```